examples/truffle/yarn.lock

```diff
@@ -2287,13 +8808,47 @@ truffle-plugin-verify@^0.5.6:
     querystring "^0.2.1"
 
 truffle@^5.1.62:
-  version "5.1.62"
-  resolved "https://registry.yarnpkg.com/truffle/-/truffle-5.1.62.tgz#819b8280615345bf42bbba7967f2dc9d39347869"
-  integrity sha512-TWgZ3xSSb8us4ZkHEHRghJWivfewjEy7yd8nARd7EHvoOi5lcUmKnbsunP+G7u/AO47gRoEqwFdPuGPdXrBM1A==
+  version "5.2.2"
```

Comment:
ubernit: Do you mind updating `package.json` with the `5.2.2` version so it matches whats in the lock file? It just makes it easier to immediately see what the actual dependency version are. Not super important though.